### DISCUSSION TITLE
Need the option to connect using SSL for CQL Cassandra Driver

### DISCUSSION THREAD

**Discussion by ckamaleo** [JanusGraph/janusgraph#4697] [Q&A] [answered]
**Describe the feature:**
For connecting to Yugabyte CQL DB we use --ssl and provide a certfile.  Unfortunately, we don't see any option in JanusGraph to specify the certfile.

https://github.com/JanusGraph/janusgraph/blob/master/janusgraph-dist/src/assembly/cfilter/conf/janusgraph-cql.properties

**Describe a specific use case for the feature:**
Can we add the Feature to Support CQL SSL Connections?

**Comment by porunov** [comment, marked as answer]
Please, check options under ‘storage.cql.ssl’ namespace: https://docs.janusgraph.org/configs/configuration-reference/
